### PR TITLE
Change routing for pages & fix the spinner

### PR DESCRIPTION
**Pull request by AlexKapic**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -30,6 +31,7 @@
     "typescript": "4.1.5"
   },
   "devDependencies": {
+    "@types/is-uuid": "^1.0.0",
```
Comment (aleksandr-kuharenko) [inline frontend/package.json:34]:
```suggestion
    "@types/is-uuid": "1.0.0",
```

File: frontend/src/components/toolbar/components/page-item/page-item.tsx [hunk 2]
```diff
@@ -40,7 +40,13 @@ const PageItem: React.FC<Props> = ({ title = 'default', id, childPages }) => {
...
               <Accordion.Header className={styles.accordionHeader}>
                 <div className={getAllowedClasses('d-flex w-100 justify-content-between align-items-center', styles.pageItem)}>
-                  <Link onClick={(): Promise<void> => getPageById(id)} to={AppRoute.PAGES} className={getAllowedClasses(styles.navbarBrand, styles.navbarLinkInsideSection, 'd-flex')}>{title}</Link>
+                  <Link
+                    onClick={(): Promise<void> => getPageById(id)}
+                    to={ `${AppRoute.PAGE.slice(0, AppRoute.PAGE.length - 3)}${id}` as AppRoute }
+                    className={getAllowedClasses(styles.navbarBrand, styles.navbarLinkInsideSection, 'd-flex')}
+                  >
+                    {title}
+                  </Link>
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/toolbar/components/page-item/page-item.tsx:49]: this part is duplicated

File: frontend/src/components/pages/components/page-content/page-content.tsx [hunk 3]
```diff
@@ -35,7 +57,7 @@ const PageContent: React.FC = () => {
 
   return (
     <>
-      {currentPage ? <Content />
+      {!isSpinner ? <Content />
```
Comment (aleksandr-kuharenko) [inline frontend/src/components/pages/components/page-content/page-content.tsx:60]: `currentPage` was ok, why you changed it?

Comment (AlexKapic) [reply frontend/src/components/pages/components/page-content/page-content.tsx:60]: now spinner work more correctly -- it shown only during `fetch`. Previously, the spinner was tied to the presence of the current page.

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by ovchynnikov — APPROVED** [review]
Not necessary but component LinkWithTitle might be in an independent file.

**Pull request merged**